src/Runtime/jni/jnilog.c

```diff
@@ -29,6 +32,63 @@
 
 #include "jnilog.h"
 
+#define INLINE static __inline
```

Comment:
Is there a reason to use a define instead of just ```static __inline``` directly in the signatures?

Reply:
The `#define` is a reminder that `static __inline` is platform dependent and not all platforms necessarily support it. Currently all the platforms we support support it so it may seem redundant. But if we ever have to deal with a platform that doesn't support inline or uses some other directive, `INLINE` can be defined accordingly.

Reply:
I think it would be better to be explicit and use `static __inline`, so that the code is easier to read. And if there's ever a need to support a platform that does not support it, `INLINE` can be defined at that time.

Reply:
...and doing double amount of work to get what's there now. I'm not sure what's not explicit about INLINE. As mentioned I intentionally use it as a reminder and I don't see it makes the code any harder to read.

Reply:
Macros (even simple ones) require an extra step to consider when reading the code. For example, in this case when I first looked at the code, I had to go back and check how the macro was defined because I was confused that it existed at all since `static __inline` generally works.

I find that there's little value in preparing for the possibility of a use (like another platform that doesn't support `static __inline`) at the expanse of current readability, but if you have strong feelings about using the macro, that's fine too.

Reply:
See PTHREAD_ONCE comment below.